### PR TITLE
Fix dockerfile

### PR DESCRIPTION
**Pull request by ostcar**

### CODE REVIEW

File: pkg/manage/cmd_check_server.go [hunk 1]
```diff
@@ -69,9 +69,10 @@ func (s *Server) CheckServer(ctx context.Context, in *proto.CheckServerRequest)
...
 	addr := net.JoinHostPort(host, port)
 	d := net.Dialer{}
-	_, err := d.DialContext(ctx, "tcp", addr)
+	con, err := d.DialContext(ctx, "tcp", addr)
 	for err != nil && ctx.Err() == nil {
 		time.Sleep(100 * time.Millisecond)
-		_, err = d.DialContext(ctx, "tcp", addr)
+		con, err = d.DialContext(ctx, "tcp", addr)
 	}
+	con.Close()
```
Comment (normanjaeckel) [inline pkg/manage/cmd_check_server.go:77]: This might crash if err is not nil but ctx.Err() is also not nil.

File: pkg/manage/server.go [hunk 2]
```diff
@@ -108,3 +116,20 @@ func (c *ServerConfig) DatastoreWriterURL() url.URL {
...
 	return u
 }
+
+// waitForShutdown blocks until the service exists.
+//
+// It listens on SIGINT and SIGTERM. If the signal is received for a second
+// time, the process is killed with statuscode 1.
+func waitForShutdown() {
+	sigint := make(chan os.Signal, 1)
+	// syscall.SIGTERM is not pressent on all plattforms. Since the autoupdate
```
Comment (normanjaeckel) [inline pkg/manage/server.go:126]: Fix typo pls. We do not have autoupdate here.

File: pkg/manage/server.go [hunk 3]
```diff
@@ -19,14 +22,19 @@ func RunServer(cfg *ServerConfig) error {
...
 
-	s := grpc.NewServer()
-	proto.RegisterManageServer(s, newServer(cfg))
+	srv := grpc.NewServer()
+	proto.RegisterManageServer(srv, newServer(cfg))
 
-	fmt.Printf("Running manage service on %s\n", addr)
+	go func() {
+		waitForShutdown()
+		srv.GracefulStop()
```
Comment (normanjaeckel) [inline pkg/manage/server.go:30]: Is it possible that the process exits regularly (in the main()) before this is done?

File: pkg/manage/cmd_check_server.go (hunk 1 above)
Comment (ostcar) [reply pkg/manage/cmd_check_server.go:77]: Why do you think that?

File: pkg/manage/server.go (hunk 2 above)
Comment (ostcar) [reply pkg/manage/server.go:126]: done

File: pkg/manage/server.go (hunk 3 above)
Comment (ostcar) [reply pkg/manage/server.go:30]: No. Why do you think that?

File: pkg/manage/cmd_check_server.go (hunk 1 above)
Comment (ostcar) [reply pkg/manage/cmd_check_server.go:77]: Ahh, know I see. Done

**Pull request merged**